web/packages/teleterm/src/ui/ConnectMyComputer/NavigationMenu.tsx

```diff
@@ -57,12 +56,10 @@ export function NavigationMenu(props: NavigationMenuProps) {
   );
 
   // Don't show the navigation icon for leaf clusters.
-  if (cluster.leaf) {
+  if (cluster.leaf || !canUse) {
```

Comment:
`canUse` loses context here I think but I don't have another suggestion besides some long name like `canUseConnectMyComputer` which matches the name of the memoized func. I'm ok with going with this, jsut thought I'd mention it.

Reply:
Yeah, I was thinking about this too, but it seemed to me that exporting a function `canUseConnectMyComputer` from a hook called `useConnectMyComputer` is too long. 
I could rename the function while using it (`const { canUse: canUseConnectMyComputer } = useConnectMyComputerContext()`), but since it is declared only a few lines above `if (cluster.leaf || !canUse)` I think you can easily see where `canUse` comes from.